### PR TITLE
图像识别

### PR DESCRIPTION
**Pull request by XatMassacrE**
resolve [#42](https://github.com/xitu/tensorflow/issues/42)

### CODE REVIEW

**Comment by MRNIU** [comment]
@leviding  校对认领

**Comment by leviding** [comment]
@MRNIU ok

File: tensorflow/docs_src/tutorials/image_recognition.md [hunk 1]
```diff
@@ -130,42 +108,39 @@ I tensorflow/examples/label_image/main.cc:206] academic gown (401): 0.0103579
...
+在训练过程中调节这些值来匹配你使用过的值。
 
-You can see how they're applied to an image in the
-[`ReadTensorFromImageFile()`](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/examples/label_image/main.cc#L88)
-function.
+你可以看到 [`ReadTensorFromImageFile()`](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/examples/label_image/main.cc#L88) 
+这个函数是如何应用在一张图片上的。
 
 ```C++
 // Given an image file name, read in the data, try to decode it as an image,
```
Comment (MRNIU) [inline tensorflow/docs_src/tutorials/image_recognition.md]: 请翻译注释，所有的注释都需要翻译

File: tensorflow/docs_src/tutorials/image_recognition.md [hunk 2]
```diff
@@ -418,41 +384,36 @@ Finally we print the labels we found.
...
-read the recent work of all the papers referenced in this tutorial.
+也包含了这部分内容。
+
+
+更多的关于实现卷积神经网络的资源，你可以去 
+TensorFlow @{$deep_cnn$deep convolutional networks tutorial} 查看，
+或者跟随我们的 @{$beginners$ML beginner} 或 @{$pros$ML expert} MNIST 入门指南来
+慢慢来学习。最后，如果你想快速提升在这个领域的研究，
+你可以
+阅读这篇指南所有引用到的论文的最新的工作和进展。
```
Comment (MRNIU) [inline tensorflow/docs_src/tutorials/image_recognition.md]: 应该是引用论文的最新成果？请检查一下

File: tensorflow/docs_src/tutorials/image_recognition.md [hunk 3]
```diff
@@ -418,41 +384,36 @@ Finally we print the labels we found.
...
-[free online book](http://neuralnetworksanddeeplearning.com/chap1.html)
-is an excellent resource. For convolutional neural networks in particular,
-Chris Olah has some
-[nice blog posts](https://colah.github.io/posts/2014-07-Conv-Nets-Modular/),
-and Michael Nielsen's book has a
+这里的异常处理是使用 TensorFlow 的 `Status` 对象，`Status` 对象使用起来非常方便，因为
+它的 `ok()` 检查器可以让你知道是否有任何异常发生，并且还可以以可读的错误信息的形式
+把它们打印出来。
+
+这里我们展示了图像识别，但是你应该能够在各种各样的领域中以及任何你发现或者你自己训练的模型中使用这些
```
Comment (MRNIU) [inline tensorflow/docs_src/tutorials/image_recognition.md]: object在这里应该没有图像的意思吧？

**Comment by MRNIU** [comment]
@XatMassacrE @leviding 校对完成，请查看

**Comment by leviding** [comment]
@XatMassacrE 修改一下吧

**Comment by XatMassacrE** [comment]
@MRNIU 感谢校对。
@leviding 已按照校对者校对意见修改完毕。

File: tensorflow/docs_src/tutorials/image_recognition.md [hunk 4]
```diff
@@ -384,12 +353,12 @@ Load, resize, and process the input image.
...
 ```
-Here we run the loaded graph with the image as an input.
+这里我们运行使用图片作为输入的已经载入的 graph。
 
 ```C++
-  // This is for automated testing to make sure we get the expected result with
-  // the default settings. We know that label 866 (military uniform) should be
-  // the top label for the Admiral Hopper image.
+  // 这里是为了保证我们在默认设置下得到了我们想要结果的自动测试程序。
+  // 我们知道标签 866 (军装）应该
```
Comment (leviding) [inline tensorflow/docs_src/tutorials/image_recognition.md]: 这一行中英文括号没换全

File: tensorflow/docs_src/tutorials/image_recognition.md [hunk 5]
```diff
@@ -1,127 +1,107 @@
...
 [Inception-v3]: https://arxiv.org/abs/1512.00567
 
-Inception-v3 is trained for the [ImageNet] Large Visual Recognition Challenge
-using the data from 2012. This is a standard task in computer vision,
-where models try to classify entire
-images into [1000 classes], like "Zebra", "Dalmatian", and "Dishwasher".
-For example, here are the results from [AlexNet] classifying some images:
+Inception-v3 从 2012 年就开始使用数据针对 [ImageNet] 这个
+大型视觉挑战任务训练了。
+将所有的图片分成像"斑马"，"达尔马西亚狗"，"洗碗工"等 [1000 个类别] 是计算机视觉领域的一个标准任务，
```
Comment (leviding) [inline tensorflow/docs_src/tutorials/image_recognition.md]: 英文双引号换成中文双引号

File: tensorflow/docs_src/tutorials/image_recognition.md [hunk 6]
```diff
@@ -130,54 +110,51 @@ I tensorflow/examples/label_image/main.cc:206] academic gown (401): 0.0103579
...
+在这种情况下，我们使用默认图片 [Admiral Grace Hopper](https://en.wikipedia.org/wiki/Grace_Hopper)，
+你可以看到网络使用 0.8 的高分正确的标识出了她穿的是军装。
 
 
 <div style="width:45%; margin:auto; margin-bottom:10px; margin-top:20px;">
   <img style="width:100%" src="https://www.tensorflow.org/images/grace_hopper.jpg">
 </div>
 
-Next, try it out on your own images by supplying the --image= argument, e.g.
+下面你可以通过 --image= 参数来检测一张自己的图片试试，例如：
```
Comment (leviding) [inline tensorflow/docs_src/tutorials/image_recognition.md]: --image= argument 保留

File: tensorflow/docs_src/tutorials/image_recognition.md [hunk 7]
```diff
@@ -130,54 +110,51 @@ I tensorflow/examples/label_image/main.cc:206] academic gown (401): 0.0103579
...
 
 ```bash
 bazel-bin/tensorflow/examples/label_image/label_image --image=my_image.png
 ```
 
-If you look inside the [`tensorflow/examples/label_image/main.cc`](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/examples/label_image/main.cc)
-file, you can find out
-how it works. We hope this code will help you integrate TensorFlow into
-your own applications, so we will walk step by step through the main functions:
+如果你仔细浏览 [`tensorflow/examples/label_image/main.cc`](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/examples/label_image/    main.cc)
```
Comment (leviding) [inline tensorflow/docs_src/tutorials/image_recognition.md]: 看后面链接，多了好多空格

**Comment by leviding** [comment]
@XatMassacrE 有几个格式问题，修改下吧，然后把全文不应该断行的地方都调整下，就可以 merge 了

**Comment by XatMassacrE** [comment]
@leviding 感谢校对，已按照校对意见修改完毕。
